docs/Testing.md

```diff
@@ -31,6 +31,6 @@ This project has three types of tests:
 1. *Unit tests* targeting the lowest level methods/classes.<br>
    e.g. `seedu.address.commons.StringUtilTest`
 1. *Integration tests* that are checking the integration of multiple code units (those code units are assumed to be working).<br>
-   e.g. `seedu.address.storage.StorageManagerTest`
+   e.g. `storage.spleetwaise.address.StorageManagerTest`
 1. Hybrids of unit and integration tests. These test are checking multiple code units as well as how the are connected together.<br>
-   e.g. `seedu.address.logic.LogicManagerTest`
+   e.g. `logic.spleetwaise.address.LogicManagerTest`
```

Comment:
This might have been a mistake during the reformatting. Should it be `spleetwaise.address.*` instead? Similar issue for the other docs.